pkg/controllers/verifier_controller.go

```diff
@@ -76,13 +73,7 @@ func (r *VerifierReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
-	namespace, err := getCertStoreNamespace(req.Namespace)
-	if err != nil {
-		verifierLogger.Error(err, "unable to get default namespace for certstore specified in verifier crd")
-		return ctrl.Result{}, err
-	}
-
-	if err = verifierAddOrReplace(verifier.Spec, resource, namespace); err != nil {
+	if err := verifierAddOrReplace(verifier.Spec, resource, constants.EmptyNamespace); err != nil {
```

Comment:
We won't need the namespace var since now. In the future, verifier will just keep the original configurations from users, like the ref to certStore/KMP. For the logics matching reference to certStore/KMP, it's moved to certStore/KMP implementation so that we could have different behavior on them. And it would be easier to manage it than splitting namespace fetching, namespace prefix and matching into different places.